src/parser.rs

```diff
@@ -1854,20 +1847,17 @@ impl Parser {
     }
 
     /// Parse a LIMIT clause
-    pub fn parse_limit(&mut self) -> Result<Option<Expr>, ParserError> {
+    pub fn parse_limit(&mut self) -> Result<Option<u64>, ParserError> {
```

Comment:
I'm not against this change, but I didn't make it sooner because at least [Postgres allows expressions here](https://www.postgresql.org/docs/11/sql-select.html#SQL-LIMIT).

Reply:
Oh, good point. I've backed this bit out.